Iteration 1

kyle1373 opened the pull request:
- [x] Menu Bar
- [x] About Section
- [x] FAQ Section (and Background in one branch)
- [x] Fix display bug on IOS
- [x] Social media icons

FINAL FIXES:

- [x] Title centering issue on Safari
- [x] Move geisel background to the left more to align better with falling star background image
- [x] Scale falling star and geisel graphics according to screen size. If the screen is larger, make the images larger
- [x] Use Lexend Peta font for questions in the FAQ section. Everything else will be Poppins font.
- [x] Update FAQ text (in https://docs.google.com/document/u/3/d/1tJ5ha2aky7Bq3RXZfErd2w6q2t15vGD3XqEhL_MTgn8/edit)
- [x] Put instagram href as https://www.instagram.com/hardhackucsd/
- [x] Hard hack won't have sponsors, but it is not confirmed atm. We'll have the code for sponsors in case it happens.
- [x] Have padding of left side of the screen to top left logo be the same as FAQ menu button to right side of the screen 
- [x] In the drop down menu on mobile, add more padding under the FAQ

Adneths committed:
Fix fallenstar animation

Adneths committed:
Shift button down

Adneths committed:
Added menu bar

Adneths committed:
Fix icon desync

kyle1373 committed:
Merge pull request #6 from HKN-UCSD/bill/feature/menu-bar

Added menu bar

Adneths committed:
Update index.js

Adneths committed:
Merge branch 'release/menu-about-faq' into bill/fix/background

Adneths committed:
Revert "Merge branch 'release/menu-about-faq' into bill/fix/background"

This reverts commit 288a531e40de6fffba6ad16658f4297f7b02beee, reversing
changes made to 236ceccc7a08c3d0f3b512b8d8faad5b8623443f.

Adneths committed:
Manually adding menu bar

Because git merging is not working as intended for some reason.

Adneths committed:
Merge pull request #5 from HKN-UCSD/bill/fix/background

Fix fallenstar animation & Manually added menu bar

Adneths committed:
Detect window size on refresh

Adneths committed:
Actually detect window size

Adneths committed:
Linked FAQ menu button

Adneths commented:
The bug on ios appears to be due to the fact that overflow: hidden and overflow: clip is not supported on older versions of safari. Using a newer device the page displays as intended. I'm not sure what should we do about this though.

Adneths committed:
Tweaked page layout

Adneths committed:
Change animation duration

Adneths committed:
Added poppins font

Adneths committed:
Try fix ios bug

Adneths committed:
Fix fallenstar position

Adneths committed:
Fit apply button on screen

Adneths committed:
Update dropdown.js

Increase "hidden" threshold

ryanyychen committed:
Added social icons

ryanyychen committed:
Replaced icons with proper svg

Adneths committed:
Change dropdown

Uses attributes to store open status

Adneths committed:
Remove extra div in dropdown-box

andrewzpu committed:
Added About Section

ryanyychen committed:
Added flex for social icons

kyle1373 commented on assets/modules/dropdown/dropdown.css:
```diff
@@ -0,0 +1,64 @@
+	font-weight: 700;
+	font-size: 24px;
+}
+dropdown-box .header svg {
```
There is a problem with dropdownbox

Adneths committed:
Fix bug on safari

Hide text with opacity 0 when closed

andrewzpu committed:
Changed Padding Spacing

ryanyychen committed:
Use em instead of px for icon size

andrewzpu committed:
Added Links to Past Projects Buttons

andrewzpu committed:
Adjusted Spacing

andrewzpu committed:
Left Align

ryanyychen committed:
Remove href links

ryanyychen committed:
Add max-width to icons and padding on sides

ryanyychen committed:
Removed Facebook and Discord icons

ryanyychen committed:
Changed vertical spacing of icons

ryanyychen committed:
Increase padding below icons

ryanyychen committed:
Adjusted icon size

ryanyychen committed:
Adjust icon size to change with vw

ryanyychen committed:
Typo in min-width lol

ryanyychen committed:
Adjusted email icon size

ryanyychen committed:
Updated footer to new design

ryanyychen committed:
Adjust icon centering and text width on phone

ryanyychen committed:
Readjust phone text width

ryanyychen committed:
Adjust email icon size

ryanyychen committed:
Fixed icon centering

ryanyychen committed:
Merge pull request #9 from HKN-UCSD/ryan/feature/socials

Ryan/feature/socials

ryanyychen committed:
Removed unnecessary social icons

ryanyychen committed:
Removed the wrong svg file, fixed it

kyle1373 committed:
Merge pull request #10 from HKN-UCSD/andrew/feature/about-section

Andrew/feature/about section

kyle1373 commented:
Final things:

- [ ] Move geisel background to the left more to align better with falling star background image

Scale falling star and geisel graphics according to screen size. If the screen is larger, make the images larger

Use Lexend Peta font for questions in the FAQ section. Everything else will be Poppins font.

Update FAQ text (in https://docs.google.com/document/u/3/d/1tJ5ha2aky7Bq3RXZfErd2w6q2t15vGD3XqEhL_MTgn8/edit)

Put social media icon for email as hardhack@eng.ucsd.edu. Instagram is TBD. There won't be any other icons

Hard hack won't have sponsors, but it is not confirmed atm. We'll have the code for sponsors in case it happens.

Have padding of left side of the screen to top left logo be the same as FAQ menu button to right side of the screen

Adneths committed:
Limit Dropdown min size

Adneths committed:
Create and add section heading element

Adneths committed:
Merge branch 'release/menu-about-faq' of https://github.com/HKN-UCSD/hard-hack-2022 into release/menu-about-faq

Adneths committed:
Speed up img animation

Adneths committed:
Fix fonts

Adneths committed:
Link about-page to menubar

Adneths committed:
Update FAQ

Adneths committed:
Update index.css

Adneths committed:
Match menubar right pad with logo left pad

Adneths committed:
Increase padding in dropdown menubar

Adneths committed:
Scale library with screen

Adneths committed:
Scale fallenstar with screen

Adneths committed:
Fix fallenstar above dropdown menubar

Adneths committed:
Fix dropdown

Header text causes arrow to misalign if screen too thin. Fixed by scaling down header font.

ryanyychen committed:
Added Instagram icon href

ryanyychen committed:
Use float in css instead of align intext

ryanyychen committed:
Revert to using align in html

kyle1373 committed:
Added google searchability
